modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/BindConfigurationMVCActionCommand.java

```diff
@@ -206,6 +209,28 @@ protected void configureTargetService(
 					ConfigurationModel.PROPERTY_VALUE_COMPANY_ID_DEFAULT);
 			}
 
+			if (configurationModel.isFactory()) {
+				String pid = configuration.getPid();
+
+				int index = pid.lastIndexOf('.');
+
+				String factoryPid = pid.substring(index + 1);
+
+				StringBundler sb = new StringBundler(7);
+
+				sb.append("file:");
+				sb.append(PropsUtil.get(PropsKeys.LIFERAY_HOME));
+				sb.append(ConfigurationModel.CONFIGURATION_FOLDER);
```

Comment:
These two lines (222-223) can be replaced with`PropsKeys.MODULE_FRAMEWORK_CONFIGS_DIR`.